src/Core/AdminConsole/Models/Data/EventIntegrations/IntegrationListenerConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Bit.Core.Enums;
+
+namespace Bit.Core.AdminConsole.Models.Data.EventIntegrations;
+
+public interface IntegrationListenerConfiguration : EventListenerConfiguration
```

Comment:
... and maybe use the `I` throughout. We also often have interfaces and implementations in separate folders, but not sure if I love that pattern.